server/templates/custom_dc/floret/homepage.html

```diff
@@ -0,0 +1,228 @@
+{#
+  Copyright 2023 Google LLC
+ 
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+       http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ #}
+ 
+ {%- extends BASE_HTML -%}
+ 
+ {% set main_id = 'homepage' %}
+ {% set page_id = 'page-homepage' %}
+ 
+ {% block head %}
+ <link rel="stylesheet" href={{url_for('static', filename='css/homepage.min.css', t=config['VERSION'])}}>
+ <script src={{url_for('static', filename='homepage.js', t=config['VERSION'])}} async></script>
+ <script src="http://localhost:8080/datacommons.js"></script>
```

Comment:
is this localhost?

Reply:
Fixed-changed to dev.datacommons.org for now (until the next prod push)